packages/amplify-graphql-auth-transformer/src/__tests__/__snapshots__/amplify-admin-auth.test.ts.snap

```diff
@@ -4,8 +4,7 @@ exports[`admin roles should be return the field name inside field resolvers 1`]
 "## [Start] Field Authorization Steps. **
 #set( $isAuthorized = false )
 #if( $util.authType() == \\"IAM Authorization\\" )
-  #set( $adminRoles = [\\"us-fake-1_uuid_Full-access/CognitoIdentityCredentials\\",\\"us-fake-1_uuid_Manage-only/CognitoIdentityCredentials\\"] )
```

Comment:
Is it possible to add unit tests to verify the admin roles are added to stash as expected since they are being removed from old snapshots? Looks like they are moved to the inline statement of `before` stage.

Reply:
I've updated and added positive/negative stash checks to these units.